src/components/ShareStructure/ShareStructure.vue

```diff
@@ -38,20 +38,23 @@
       </div>
     </template>
 
-    <v-expand-transition>
-      <v-card flat class="add-share-structure-container" v-if="showAddShareStructureForm">
-        <edit-share-structure
-          :initialValue="currentShareStructure"
-          :activeIndex="activeIndex"
-          :shareId="shareId"
-          :parentIndex="parentIndex"
-          :shareClasses="shareClasses"
-          :resolution-required="resolutionRequired"
-          @addEditClass="addEditShareClass($event)"
-          @resolutionPrompt="emitResolutionPrompt($event)"
-          @resetEvent="resetData()"/>
-      </v-card>
-    </v-expand-transition>
+    <div :class="{'invalid-section': invalidSection}">
```

Comment:
Instead of adding a div, did you try adding the class to the v-card element?

Reply:
I did yea. I also tried at the root level of the edit form, but it had too many implications with the different means of edit. ( ie editing share class vs share series vs New share class ... etc etc.

So i applied them individually to the cells or in this case, created a small wrapper.

Reply:
OK.